tests/Jellyfin.Api.Tests/Jellyfin.Api.Tests.csproj

```diff
@@ -15,6 +19,13 @@
 
   <ItemGroup>
     <ProjectReference Include="../../MediaBrowser.Api/MediaBrowser.Api.csproj" />
+    <ProjectReference Include="..\..\Jellyfin.Api\Jellyfin.Api.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Reference Include="Microsoft.AspNetCore.Authentication, Version=3.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60">
```

Comment:
Why? This shouldn't be needed since `Jellyfin.Api` already includes it.

Reply:
Not sure what happened there - IDE must have referenced it automatically as a suggestion or something. I've removed it now.